packages/std/src/addresses.rs

```diff
@@ -93,25 +166,25 @@ impl PartialEq<Addr> for String {
 
 impl From<Addr> for String {
     fn from(addr: Addr) -> Self {
-        addr.0
+        addr.as_str().to_owned()
     }
 }
 
 impl From<&Addr> for String {
     fn from(addr: &Addr) -> Self {
-        addr.0.clone()
+        addr.as_str().to_owned()
     }
 }
 
 impl From<Addr> for HumanAddr {
```

Comment:
Maybe we remove these - let's not support HumanAddr with the new code (and if someone wants that can always do `addr.as_str().into()`.